install_config/install/advanced_install.adoc

```diff
@@ -2170,102 +2170,71 @@ ifdef::openshift-origin[]
 *openshift/origin-ansible*
 endif::[]
 image is a containerized version of the {product-title} installer that runs as a
-link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux_atomic_host/7/html/managing_containers/running_system_containers[system container]. System containers are stored and run outside of the traditional
-*docker* service. Functionally, using the containerized installer is the same as
-using the traditional RPM-based installer, except it is running in a
-containerized environment instead of directly on the host.
-
-. Use the Docker CLI to pull the image locally:
-+
-----
-ifdef::openshift-enterprise[]
-$ docker pull registry.access.redhat.com/openshift3/ose-ansible:v3.6
-endif::[]
-ifdef::openshift-origin[]
-$ docker pull docker.io/openshift/origin-ansible:v3.6
-endif::[]
-----
-
-. The installer system container must be stored in
-link:https://access.redhat.com/documentation/en-us/red_hat_satellite/6.2/html/content_management_guide/managing_ostree_content[OSTree]
-instead of defaulting to *docker* daemon storage. Use the Atomic CLI to import
-the installer image from the local *docker* engine to OSTree storage:
+link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux_atomic_host/7/html/managing_containers/running_system_containers[system container].
+System containers are stored and run outside of the traditional
+*docker* service. The installer image provides the same functionality
+as the traditional RPM-based installer, but it runs in a containerized
+environment that provides all of its dependencies rather than being
+installed directly on the host.
+
+. As the root user, use the Atomic CLI to run the system container as a one-time systemd service:
 +
 ----
-$ atomic pull --storage ostree \
-ifdef::openshift-enterprise[]
-    docker:registry.access.redhat.com/openshift3/ose-ansible:v3.6
-endif::[]
-ifdef::openshift-origin[]
-    docker:docker.io/openshift/origin-ansible:v3.6
-endif::[]
-----
-
-. Install the system container so it is set up as a systemd service:
-+
-----
-$ atomic install --system \
+# atomic install --system \
```

Comment:
atomic install I think you have to run as root. Is it customary to use the root prompt in docs?

Reply:
Correct. It must be run as root.

Reply:
yes, it must run as root. "atomic install --system" would fail anyway if not running as root.